src/hf_hydrodata/model/dataset_version.csv

```diff
@@ -0,0 +1,4 @@
+id,title
+"",No dataset version
+0.9,CW3E version before Aug 2024
+1.0,CW3E with corrected temps
```

Comment:
Perhaps have the "title" be more generic like "Version 0.9 of the dataset" and "Version 1.0 of the dataset" instead of specific CW3E language.